crates/runtime/src/dataconnector.rs

```diff
@@ -250,6 +257,7 @@ pub async fn register_all() {
     register_connector_factory("motherduck", motherduck::MotherDuck::create).await;
     #[cfg(feature = "clickhouse")]
     register_connector_factory("clickhouse", clickhouse::Clickhouse::create).await;
+    register_connector_factory("graphql", graphql::GraphQL::create).await;
```

Comment:
we probably should move non-feature toggled things into the top. Not sure what particular order here we are following.

Reply:
Agree, will make a PR for this